Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    [SerializeField]
+    private string scenetoLoad;
+
+    private bool isPlayerInTrigger;
+
+    // cant use this bc it triggers the door twice, bc the player has two colliders
```

Comment:
If it's not going to be used in the code anymore I'd save this elsewhere for later.